codalab/lib/download_manager.py

```diff
@@ -89,7 +103,10 @@ def get_target_info(self, uuid, path, depth):
                         )
                     )
                 elif 'error_code' in result:
-                    raise http_error_to_exception(result['error_code'], result['error_message'])
+                    try:
+                        raise http_error_to_exception(result['error_code'], result['error_message'])
+                    except NotFoundError as e:
```

Comment:
This is the call made to the worker if the bundle is still `RUNNING`. In this case, the call may fail for a variety of reasons, but we care about the `NotFound` case, which means the worker successfully looked for the path and did not find it. Otherwise, we don't want to bother.